src/tests/ftest/util/ec_utils.py

```diff
@@ -49,6 +54,37 @@ def setUp(self):
         self.server_count = len(self.hostlist_servers) * 2
         #Create the Pool
         self.create_pool_max_size()
+        self.update_ior_cmd_with_pool()
+
+    def ec_contaier_create(self, oclass):
+        """Create the container for EC object"""
+        # Get container params
+        self.ec_container = TestContainer(
+            self.pool, daos_command=DaosCommand(self.bin))
+        self.ec_container.get_params(self)
+        self.ec_container.oclass.update(oclass)
+        # update object class for container create, if supplied
+        # explicitly.
+        data_parity = self.get_data_parity_number(oclass)
+        self.ec_container.properties.update("rf:{}".format(data_parity[1]))
```

Comment:
Is there a way to come with a mechanism like data_parity["parity"] instead of data_parity[1]? In this way, no need to access a particular array element. Not critical. Just a thought.

Reply:
sure can change to dictionary instead of a list.